src/quoridor/QuoridorPlayer.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace QuoridorEngine.src.quoridor
+{
+    /// <summary>
+    /// Class representing quoridor player
+    /// </summary>
+    internal class QuoridorPlayer
+    {
+        private readonly int totalWalls = 10;
+        private int availableWalls;
+        private int victories = 0;
```

Comment:
Not sure we need those 3 statistics atm especially since we haven't yet decided whether the Player objects will be persistent.

For example when a game is over it might be easier to just throw away the Board and Player classes and create new ones instead of resetting everything by hand. We will be doing this only once in a while anyway so the performance cost is negligible.

 Also the concept of an in game player (Black or white) is not tied to the concept of the physical player. For instance, I might play multiple games, some as white some as black. Therefore this class might not be the right place to keep track of those stats anyway